### PR TITLE
[4.0] Styling for com_search

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

Some styling updates for com_search
This PR also addresses some of the form UI updates introduced into BS4-beta.3

### CODE REVIEW

**Comment by ghost** [comment]
@C-Lodder can you please describe what styling changes are tester have to look for or is this a Code review?

**Comment by C-Lodder** [comment]
Just ensuring the forum ties in with Bootstrap as opposed to unstyled elements. Can be a code review

**Comment by wilsonge** [comment]
We need to fix travis tests here before it gets merged too

**Comment by C-Lodder** [comment]
@wilsonge done

**Comment by C-Lodder** [comment]
@wilsonge conflicts done. Once merged I'll fix the other PR's

File: Gruntfile.js [hunk 1]
```diff
@@ -759,7 +759,7 @@ module.exports = function(grunt) {
 	grunt.registerTask('compile:site', 'Compiles the stylesheets files for the site template', function() {
 		grunt.task.run([
 			'uglify:templates',
-			'scsslint:site',
+			//'scsslint:site',
```
Comment (wilsonge) [inline Gruntfile.js]: Why is this commented out (here and for admin?)

Comment (C-Lodder) [reply Gruntfile.js]: ah sorry. had issues downloading the lint package on the newest version of Ruby. Will fix in a min

File: modules/mod_search/tmpl/default.php [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 ?>
 <div class="search">
-	<form action="<?php echo JRoute::_('index.php'); ?>" method="post">
+	<form action="<?php echo JRoute::_('index.php', true); ?>" method="post">
```
Comment (wilsonge) [inline modules/mod_search/tmpl/default.php]: What's the need for the true param here?

**Comment by wilsonge** [comment]
com_search seems to be broken before and after this patch (it misses the `option=com_search` from the URL) but after manually adding that into the url, things work, and styling does look better after applying this patch. So merging this and going to create an issue to figure out why search isn't working

**Pull request merged**